content/en-US/html/reference/elements/input/input.md

```diff
@@ -20,9 +20,9 @@ attributes:
         - ../input/attributes/value.md
     global: /content/html/global_attributes
 examples:
-    - examples/Examples_overview
-    - examples/Styling_input_elements
-    - examples/Custom_error_messages
+    - examples/Examples_overview.md
```

Comment:
A funny thing here: `Examples_overview.md` doesn't actually contain an example. Should this exist? It seems to me that an example that doesn't contain any code blocks is dubious. Or is this a linting issue?

Reply:
Yes, I think you're right.

This is partly about how much flexibility we should give to writers. In this case they want to just point to another place people can find examples, and this is an easy way to provide that.

But anyone trying to consume these examples *as examples*, rather than as bits of a page, is going to be confused by this. So I agree that we should not do this, and a principle for designing linting rules should be that the examples should contain examples :), even if they are not always executable.

I'm not sure how we should address this desire for flexibility. Perhaps it's not that important. Perhaps writers should always include actual examples (for example, here, instead of vaguely gesturing at some other pages, this page could include some actual examples living under, say, /input/button). Or perhaps we should explicitly represent some "preamble" prose that can be added at the start of an **Examples** section, to give writers that flexibility.

But let's not add more stuff until we're sure we need it. For now I've removed this "example" from the content (actually, this page has been completely rewritten and now contains no examples at all!).